src/ol/render/canvas/replaygroup.js

```diff
@@ -130,20 +204,30 @@ ol.render.canvas.ReplayGroup.prototype.forEachFeatureAtCoordinate = function(
     ol.extent.buffer(hitExtent, resolution * this.renderBuffer_, hitExtent);
```

Comment:
Yes that seems to be true, as all draw instructions not intersecting the hitContext are skipped in the replay. Normally this would not happen as the the renderBuffer defaults to 100px and therefore is much greater than the hitTolerance.

I add this.

Seeing here that the hitBuffer is only multiplied with the normal resolution i will apply the same method to the extent of the vectortiles.